src/newSeed/baseStage.scss

```diff
@@ -127,20 +139,7 @@
       }
     }
   }
-  &.stage2 {
```

Comment:
why are you deleting stage2?

Reply:
The styles on stage2 are already in the global styles as such it's styles are uneeded